pkg/cmd/roachtest/tests/ruby_pg.go

```diff
@@ -87,6 +87,7 @@ func registerRubyPG(r registry.Registry) {
 			"install ruby 3.1.2",
 			`mkdir -p ruby-install && \
         curl -fsSL https://github.com/postmodern/ruby-install/archive/v0.8.3.tar.gz | tar --strip-components=1 -C ruby-install -xz && \
+        sudo rm -rf /usr/local/bin/* && \
```

Comment:
Is it safe to delete everything instead of targeting specific programs?

Reply:
yes, a fresh roachprod instance does not have anything in this directory